src/pages/trading-specification/components/_popup_menu.tsx

```diff
@@ -74,41 +84,46 @@ const PopUpMenu = ({ market, toggle }: TPopUpMenuProps) => {
                     <ModalCard>
                         <StyledHeading>
                             <Header type="paragraph-1" align="center" as="p">
-                                <Localize translate_text={dlTitle} />
+                                <Localize translate_text={modalTitle} />
                             </Header>
                             <CloseIconButton src={CloseIcon} onClick={toggle} />
                         </StyledHeading>
 
                         <Header type="paragraph-2" align="center" weight="normal" as="p">
-                            <Localize translate_text={dl_description} />
+                            <Localize translate_text={modalDescription} />
                         </Header>
-                        <DLTableContainer>
-                            <TableWrapper>
-                                <Tr>
-                                    <Th>
-                                        <Localize translate_text={from_lots} />
-                                    </Th>
-                                    <Th>
-                                        <Localize translate_text={to_lots} />
-                                    </Th>
-                                    <Th>
-                                        <Localize translate_text={leverage} />
-                                    </Th>
-                                </Tr>
-                                {markets_data.map((data, index) => (
-                                    <Tr key={index}>
-                                        <Td>{data.from}</Td>
-                                        <Td>{data.to}</Td>
-                                        <Td>{data.leverage}</Td>
-                                    </Tr>
-                                ))}
-                            </TableWrapper>
-                        </DLTableContainer>
-                        <HowItIsCalculated>
-                            <StyledLinkButton flat onClick={toggleCalculated}>
-                                <Localize translate_text={how_dl_is_calculated} />
-                            </StyledLinkButton>
-                        </HowItIsCalculated>
+
+                        {popupType === 'dl' && (
```

Comment:
better to use  ternary operator and  put null after

Reply:
{popupType === 'dl' ?(....):(null)

Reply:
https://medium.com/geekculture/stop-using-for-conditional-rendering-in-react-a0f7b96200f8